src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +12,8 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static int kIntakePower = 5;
```

Comment:
Intake Power is between 0 - 1, so this needs to be changed
```suggestion
    public static double kIntakePower = something;
```